domains/pallets/domain-transaction-fees/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "pallet-domain-transaction-fees"
```

Comment:
IMO, `pallet-operator-rewards` is more appropriate. We also use this pallet to store the XDM rewards.